packages/app/src/cli/models/extensions/extension-instance.ts

```diff
@@ -133,21 +123,8 @@ export class ExtensionInstance<TConfiguration extends BaseConfigType = BaseConfi
     }
   }
 
-  isDraftable(unifiedDeployment: boolean) {
-    if (unifiedDeployment) {
-      return !this.isThemeExtension
-    } else {
-      return !this.isPreviewable && !this.isThemeExtension && !this.isFunctionExtension
-    }
-  }
```

Comment:
I think we need to keep this (without passing any value) just for theme extensions

Reply:
I'll put it back 👍